### DISCUSSION TITLE
How to pass third parameter in useInfiniteQuery?

### DISCUSSION THREAD

**Discussion by qkreltms** [TanStack/query#860] [General] [answered]
If I have third parameter query in exampleApi which is used as queryFn of useInfiniteQuery, how to pass parameter?

```
const exampleApi = (_: string, cursor: number, query: { id: string }) => {
  console.log(_, cursor);
  return  axios.post(`/list${id}/${cursor || 1}`);
};
```

https://codesandbox.io/s/react-query-useinfinitequery-be62c?file=/src/App.tsx

**Comment by larryyangsen** [comment, marked as answer]
Related to: https://github.com/tannerlinsley/react-query/issues/307
